### ISSUE TITLE
Matrics in evaluation are lots of zeros.

### ISSUE THREAD

**Issue by Kai00757132** [MIC-DKFZ/nnUNet#1668]
Hi Fabian,
     I have pretrained and fine-tuned on two different datasets.Then I run prediction on the second dataset test case and run nnUNetv2_evaluate_folder to get the evaluation between prediction output and the ground truth.But my summary.json got weird values.

The prediction code I run:

`nnUNetv2_predict -i /home/user/Documents/nnUNetv2/predict/input -o /home/user/Documents/nnUNetv2/predict/output -d 504 -c 3d_fullres --save_probabilities
`
The evaluation code I run:

`nnUNetv2_evaluate_folder /home/user/Documents/dataset/Abdomen/WORD_Abdomen/labelsVal3d /home/user/Documents/nnUNetv2/predict/output -djfile /home/user/Documents/nnUNetv2/predict/output/dataset.json -pfile /home/user/Documents/nnUNetv2/predict/output/plans.json`

The output summary.json:

![image](https://github.com/MIC-DKFZ/nnUNet/assets/74966819/d2daf3d6-b368-4b61-9e39-b39ca94e6c82)

![image](https://github.com/MIC-DKFZ/nnUNet/assets/74966819/a36168e7-bdc7-4d33-aa8e-0bde79535d6e)

![image](https://github.com/MIC-DKFZ/nnUNet/assets/74966819/ca66653f-d405-4a8c-8b29-cb75bacbf210)

Thanks!

Best, Kai

**Comment by lisakausch** [comment]
Hi @Kai00757132 , 

did you compare the labels visually? Was the prediction successful?

Best,
Lisa

**Comment by Kai00757132** [comment]
Hi @lisakausch ,

The prediction was unseccessful.After that,I remake my prediction input(Because my data are 2D and overlap slices to 3D on z axis),and then it works now.So maybe the problem was because something wrong in the prediction input data.

Thaks for your concern!

Best,
Kai
